src/lite.h

```diff
@@ -129,8 +129,9 @@ char   *which      (const char *cmd);
 int     whichp     (const char *cmd);
 
 #ifndef touch
-#include <sys/stat.h>		/* utimensat() */
-#include <sys/time.h>		/* utimensat() on *BSD */
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <utime.h>
 /**
```

Comment:
I need to check this with all the supported targets, but otherwise tentatively approve.